src/Uniphar.Dotnet.Cli/Commands/GetTokenCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Uniphar.Dotnet.Cli.Token;
+
+internal sealed class GetTokenCommand : AsyncCommand<GetTokenCommand.Settings>
+{
+    public sealed class Settings : CommandSettings
+    {
+        [CommandArgument(0, "<tenantId>")]
+        public required Guid TenantId { get; set; }
+
+        [CommandArgument(1, "<clientId>")]
+        public required Guid ClientId { get; set; }
+
+    }
+
+    private const string Scope = ".default";
+
+    public async override Task<int> ExecuteAsync(CommandContext context, Settings settings)
+    {
+        var tenantId = settings.TenantId.ToString();
```

Comment:
You already use `settings.TenantId` and `settings.ClientId` directly below, you don't need these locals :)